AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_BloomAdded.py

```diff
@@ -145,40 +366,569 @@ def AtomEditorComponents_Bloom_AddedToEntity():
         Report.result(Tests.bloom_enabled, bloom_component.is_enabled())
 
         # 8. Enable the "Enable Bloom" parameter.
-        bloom_component.set_component_property_value(AtomComponentProperties.bloom('Enable Bloom'), True)
+        bloom_component.set_component_property_value(AtomComponentProperties.bloom(
+            'Enable Bloom'), True)
+        general.idle_wait_frames(1)
         Report.result(
             Tests.enable_bloom_parameter_enabled,
-            bloom_component.get_component_property_value(AtomComponentProperties.bloom('Enable Bloom')) is True)
+            bloom_component.get_component_property_value(AtomComponentProperties.bloom(
+                'Enable Bloom')) is True)
+
+        # 9. Disable the "Enable Bloom" parameter (re-enable after test for game mode verification).
+        bloom_component.set_component_property_value(AtomComponentProperties.bloom(
+            'Enable Bloom'), False)
+        general.idle_wait_frames(1)
```

Comment:
There a lot of `general.idle_wait_frames(1)` calls in this script, do we really need a `general.idle_wait_frames(1)` call after every single property set call ? This runs the risk of increasing total test run time substantially, becoming more noticeable when we add more tests in our set of tests.

If we need it, we need it, but aren't there a lot of examples of us not needing to call `general.idle_wait_frames(1)` after setting properties? In other test scripts it appears to not require it when calling `.set_component_property_value()`

Reply:
returning control to the engine for a frame is not a significant time expense. when an overall test is this long we run the risk of exceeding frame time during execution which could result in odd behavior if the engine can't execute anything.
we could probably remove some of these putting a few general.idle_wait_frames(1) in between test steps here and there.

Reply:
I've removed the majority of wait frames and placed some new waits between tests to periodically return control to the engine.